crates/vm/levm/src/precompiles.rs

```diff
@@ -197,6 +213,13 @@ pub fn is_precompile(callee_address: &Address, fork: Fork) -> bool {
         return false;
     }
 
+    cfg_if::cfg_if! {
+    if #[cfg(feature = "l2")] {
+        if RIP_PRECOMPILES.contains(callee_address) {
+            return true;
+        }
+    }}
```

Comment:
Nit: there is no need for `cfg_if` here. You can use either:
```suggestion
    #[cfg(feature = "l2")]
    if RIP_PRECOMPILES.contains(callee_address) {
        return true;
    }
```
or
```suggestion
    if cfg!(feature = "l2") {
        if RIP_PRECOMPILES.contains(callee_address) {
            return true;
        }
    }
```

See [docs for the cfg macro](https://doc.rust-lang.org/std/macro.cfg.html).

Reply:
You are right! Addressed in [08c204d](https://github.com/lambdaclass/ethrex/pull/2186/commits/08c204df7b1beba49c2743af907d368299611c53).